README.md

```diff
@@ -1,16 +1,18 @@
 # First promises
+
 Create 2 promises `promise1` and `promise2`.
+
 - `promise1` should be resolved on a `click` on `.logo`.
 - `promise2` promise should be rejected in 3 seconds.
 - add success and error handlers for both promises
 - success handler should append a `<div class="message">` with the text `Promise was resolved!` to the body
 - error handler should append a `<div class="message error-message">` with the text `Promise was rejected!` to the body
 
 ## Instructions
+
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_promise_basic_DOM/)
+   - [DEMO LINK](https://IrbisKronos.github.io/js_promise_basic_DOM/)
```

Comment:
Add this link to the PR description 
<img width="963" alt="image" src="https://github.com/mate-academy/js_promise_basic_DOM/assets/64750958/919f217f-1b3c-4cdf-9560-277876a07465">